Makefile

```diff
@@ -126,13 +126,13 @@ build: up ## Builds the :servers:game and :servers:client projects
 	$(GRADLE_TASK) :servers:game:build
 	$(GRADLE_TASK) :servers:web:build
 
-.PHONY: build-debug
-build-debug: build ## Runs the build then start the debugger socket
+.PHONY: build-game-debug
+build-game-debug: build ## Runs the build then start the debugger socket
 	$(GRADLE) java -jar -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:$(DEBUGGER_SOCKET_PORT) $(GAME_DIR)/$(DEBUGGER_BUILD_GAME_JAR_PATH)
```

Comment:
Use `$(GRADLE_TASK) :servers:game:run -Ddebug_mode="true"` instead + don't forget to use the debug plugin